docs/buildah.md

```diff
@@ -1,7 +1,7 @@
 # buildah "1" "March 2017" "buildah"
 
 ## NAME
-Buildah - A command line tool to facilitate working with containers and using them to build images.
+Buildah - A command line tool to used to build OCI Container images.
```

Comment:
Perhaps closer to the readme.md's blurb?

"A command line tool that facilitates building OCI container images."

Regardless, "to used to" needs to change to something else.